styles/variables.css

```diff
@@ -0,0 +1,32 @@
+:root {
+  /* ----- typography ----- */
+  --h1-font-size: 4rem;
+  --h2-font-size: 3rem;
+  --h3-font-size: 2.25rem;
+  --h4-font-size: 1.5rem;
+  --paragraph-font-size: 1.25rem;
+
+  /* ----- line-hight ----- */
+  --h1-line-height: 4.75rem;
+  --h2-line-height: 3.6rem;
+  --h3-line-height: 2.7rem;
+  --h4-line-height: 1.75rem;
+  --paragraph-line-height: 1.5rem;
+
+  /* ----- colors ----- */
+  --color-orange: #F6A721;
```

Comment:
What do you think if we call these colors with semantic meaning? For example. `primary`, `secondary`. Design shouldn't change from now, but you never know. Ofc white, gray and black can stay as before 😀